assignment/core/src/main/java/com/assignment/core/services/impl/AESAsymmetricCipherProvider.java

```diff
@@ -0,0 +1,42 @@
+package com.assignment.core.services.impl;
+
+import com.assignment.core.services.AsymmetricCipherProvider;
+import org.osgi.service.component.annotations.Component;
+
+import javax.crypto.Cipher;
+import javax.crypto.KeyGenerator;
+import javax.crypto.SecretKey;
+import java.util.Base64;
+
+@Component(service = AsymmetricCipherProvider.class, property = "algorithm=AES")
```

Comment:
Where is the config for this service ?
How will we get the public and private keys ?